pkg_api/utils.py

```diff
@@ -306,3 +306,102 @@ def get_query_for_conditional_get_statements(triple: Triple) -> SPARQLQuery:
 
     # Cleaning up the query
     return _clean_sparql_representation(query)
+
+
+def get_query_for_remove_preference(pkg_data: PKGData) -> SPARQLQuery:
+    """Gets SPARQL query to remove a preference.
+
+    Args:
+        pkg_data: PKG data associated to a statement.
+
+    Returns:
+        SPARQL query.
+    """
+    blank_node_id = "?statement"
+    statement = _get_statement_representation(pkg_data, blank_node_id)
+    # Remove statement description from conditions
```

Comment:
Why do we need to do that?

Reply:
The description is removed because it not possible that it is the same when asking to remove a statement, especially when using natural language. For example, a user might input "Delete my preference towards Tom Cruise" or " Remove the statements related to Tom Cruise" to remove the statement "I like Tom Cruise."